arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Anytrust orbit chains keyset generation'
+sidebar_label: 'Keyset Generation for Anytrust Orbits'
+description: 'Keyset generation and setup for Anytrust Orbit chains'
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Anytrust Orbit chains.'
+sidebar_position: 1
+---
+**Anytrust** chains, relies on an external Data Availability Committee (hereafter, "the Committee") to store data and provide it on demand, instead of using the parent chain as Data Availability (DA) layer. 
+
+The Committee comprises N members, among whom it is presumed by AnyTrust that a minimum of H members maintain integrity. H is the minimum number of trusted committee members on Anytrust chains, and it is configurable by the "assumed-honest" parameter in the keyset, and by chain owner(s). Consequently, in scenarios where `K = (N + 1) - H` members of the Committee pledge to grant access to specific data, and they must sign and attest they have the data for store to be considered successful.
+
+Each Committee member gets their own set of BLS public and private keys. It's important for every member to create their own new and secure BLS keys. They should do this on their own and make sure these keys are random and only for their use. If you need help generating BLS keys, check out our guide [here](https://docs.arbitrum.io/node-running/how-tos/running-a-daserver#generate-key).
+
+The main blockchain (parent chain) needs to know who is in the Committee. It needs to have the names of all the members and their public keys. This is important for checking and verifying information. A 'Keyset' is a list of all the public keys of the Committee members. It also shows how many signatures are needed to approve a Data Availability Certificate. This Keyset setup lets us change who is in the Committee and lets members change their keys if needed. For more details, please read the <a data-quicklook-from='inside-anytrust'>Inside Anytrust</a> page in our documentation.
+
+
+<PublicPreviewBannerPartial />
+
+In the following section, we will provide a detailed guide on the generation of a Keyset corresponding to your individual set of keys, as well as instructions for its subsequent configuration within the chain.
```

Comment:
@GreatSoshiant how do you feel about refactoring the procedural instructions out of this document, and into a dedicated how-to document?

Our [established conventions](https://www.notion.so/arbitrum/Pattern-guide-Docs-dafaf8d1ce34433d88c1ef3b4b99c19c?pvs=4#2309f7f5db284126929065ac3c56fef8) distinguish between procedural and explanatory documents to make it easier for readers to find what they're looking for.

Not a blocker, but important if we want to maintain conventions (we can always address via another PR if this is time-sensitive).

If you opt to refactor now, Orbit docs auto-generates the sidebar from the directory structure, so creating a new how-to document in the `how-tos` directory should be all that's needed. We can also link to the how-to from the concept, and vice-versa, to improve discoverability of both.

Reply:
Can we stick to this type of doc for now @symbolpunk ? We can move to the how-to doc later if you agree.